07-routing-gossip.md

```diff
@@ -524,6 +524,13 @@ makes sense to have it be a UNIX timestamp (i.e. seconds since UTC
 1970-01-01). This cannot be a hard requirement, however, given the possible case
 of two `channel_update`s within a single second.
 
+It is assumed that more than one `channel_update` message changing the channel 
+parameters in the same second may be a DoS attempt, and therefore, the node responsible 
+for signing such messages may be blacklisted. However, a node can send a same 
+`channel_update` message with a different signature (changing the nonce in signature 
```

Comment:
important to note that since ECDSA signatures are malleable, the signatures can be different even if the node only signed once.

Reply:
Very fair point. I'm incorporating this in the rationale.